rlmolecule/hallway/policy/model.py

```diff
@@ -0,0 +1,107 @@
+from typing import Optional, Tuple
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+def policy_model(hallway_size: int,
+                 features: int = 8,
+                 hidden_layers: int = 3,
+                 hidden_dim: int = 16) -> tf.keras.Model:
+
+    position_class = layers.Input([None, 1], dtype=tf.int64, name="position")
+    x = layers.Embedding(hallway_size, features, name="embedding")(position_class)
+    
+    for layer in range(hidden_layers):
+        x = layers.Dense(hidden_dim)(x)
```

Comment:
you'd probably want an activation layer (and maybe batchnorm) between these Dense layers. Although for this simple problem it likely doesn't matter. But as written, these dense layers would collapse to functionally a single matrix multiplication

Reply:
oh wow, how did i forget that... yeah i think i can just add `activation="relu"` in the Dense init